src/LendingPool.sol

```diff
@@ -144,6 +145,16 @@ contract LendingPool is Guardian, TrustedCreditor, DebtToken, InterestRateModule
         liquidationWeightTranches[index] = _weight;
     }
 
+    /**
+     * @notice Sets the maxInitiatorFee.
+     * @param maxInitiatorFee_ The maximum fee that is paid to the initiator of a liquidation
+     * @dev The liquidator sets the % of the debt that is paid to the initiator of a liquidation.
+     * This fee is capped by the maxInitiatorFee.
+     */
+    function setMaxInitiatorFee(uint88 maxInitiatorFee_) public onlyOwner {
```

Comment:
Where are the tests?

Reply:
added two for the set, other tests re maxInitFee are in vaults